plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -707,19 +707,19 @@ private static void validateOrcBloomFilterColumns(ConnectorTableMetadata tableMe
         }
     }
 
-    public static OptionalInt parseVersion(String metadataLocation)
+    public static OptionalInt parseVersion(String metadataFileName)
```

Comment:
The usages of this method in `AbstractIcebergTableOperations` should be addressed in this commit too (not in the next one).

Reply:
The first 2 commits are just cherry-picked from https://github.com/trinodb/trino/pull/18150